pype/settings/entities/schemas/projects_schema/schema_project_nuke.json

```diff
@@ -6,6 +6,32 @@
     "is_file": true,
     "children": [
         {
+            "type": "dict",
+            "collapsible": true,
+            "key": "triggers",
+            "label": "Operation triggers",
+            "children": [
+                {
+                    "type": "dict",
+                    "collapsible": false,
+                    "key": "workfile",
+                    "label": "Workfile related",
+                    "is_group": true,
+                    "children": [
+                        {
+                            "type": "boolean",
+                            "key": "open_at_start",
```

Comment:
this would probably be better suited to a global setting with host and task filtering.